.travis.yml

```diff
@@ -11,6 +12,6 @@ before_cache:
   # Tricks to avoid unnecessary cache updates
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
-script: sbt test it:test
+script: sbt +test +it:test
```

Comment:
As it is right now, test will be executed twice for each version. Check the build logs here: https://travis-ci.org/mdedetrich/stripe-scala/builds/231202498

According to this https://docs.travis-ci.com/user/languages/scala/

you should use 

```
sbt ++$TRAVIS_SCALA_VERSION test it:test
```
instead.